assets/price-per-item.js

```diff
@@ -0,0 +1,103 @@
+if (!customElements.get('price-per-item')) {
+  customElements.define(
+    'price-per-item',
+    class PricePerItem extends HTMLElement {
+      constructor() {
+        super();
+        this.input = document.getElementById(`Quantity-${this.dataset.sectionId}`);
+        this.variantId = document.getElementById(`Price-Per-Item-${this.dataset.sectionId}`).dataset.variantId;
```

Comment:
I think this is equivalent: 

```suggestion
        this.variantId = this.dataset.variantId;
```